vms/platformvm/client.go

```diff
@@ -661,7 +661,7 @@ func GetSubnetOwners(
 	return subnetOwners, nil
 }
 
-// GetDeactivationOwners returns a map of validation ID to subnet-only
+// GetDeactivationOwners returns a map of validation ID to L1
 // validation deactivation owner
```

Comment:
I feel like this just makes it read better.
```suggestion
// GetDeactivationOwners returns a map of validation ID to deactivation owners
```